### DISCUSSION TITLE
Digital space for real-time chat?

### DISCUSSION THREAD

**Discussion by decentral1se** [foodcoops/foodsoft#874] [General]
Hey, I was wondering if there is a chat room (matrix,xmpp,etc.) that people who work on improving foodsoft hang out in? 

It would be nice to have that space I think. It would make it easier to collaborate. Anyone else interested in this?

(EDIT: I am gonna post to http://foodsoft.51229.x6.nabble.com instead then I guess)

**Comment by kidhab** [comment]
I like your idea. What about a Matrix channel called foodsoft-operators?

**Reply by decentral1se** [reply]
Nice @kidhab! I made a matrix space: `#foodsoft:matrix.autonomic.zone`.

Invite link: https://matrix.to/#/#foodsoft:autonomic.zone

There is one room in the space, `#foodsoft-operators:matrix.autonomic.zone`.

Hope you can join, can make you admin.

All welcome :rocket:

**Reply by decentral1se** [reply]
People have started arriving, here is a gentle reminder to head over and join us if you operate a foodsoft instance :partying_face:

**Comment by RayOei** [comment]
For more details on chats look [here](https://forum.foodcoops.net/t/foodsoft-monthly-community-call/573)

Closing as it doesn't add much anymore.
